### ISSUE TITLE
TypeError when trying to view resource list

### ISSUE THREAD

**Issue by xrotwang** [sacrud/pyramid_sacrud#117]
Navigating to the resource list of a model, I get the following error:
```
  File "/home/robert/venvs/wals3/local/lib/python2.7/site-packages/pyramid_sacrud/templates/sacrud/crud/list.jinja2", line 73, in block "body"
    {% include "paginator.jinja2" %}
  File "/home/robert/venvs/wals3/local/lib/python2.7/site-packages/pyramid_sacrud/templates/sacrud/crud/paginator.jinja2", line 2, in top-level template code
    {{ paginator.pager(link_tag=paginator.kwargs['link_tag'])|safe}}
TypeError: pager() got an unexpected keyword argument 'link_tag'
```

**Comment by uralbash** [comment]
Please check the version of ``paginate`` in your python env. It require >=0.5.0, see CHANGELOG:

```rst

Version 0.5.0 - 2015-08-29
--------------------------

Changes:
- link_tag callable can now be passed to generate custom link markup
- page object now has link_map() method that returns a mapping of information useful for generating
  custom markup based on paginator data
```

I will fix it in requirements in new version of ``pyramid_sacrud``, but you can do it manually:

```bash
pip install paginate -U
```

**Comment by uralbash** [comment]
fixed in new 0.3.1 version!
